CommandLine/Non-modular/Maven/hellofx/pom.xml

```diff
@@ -17,25 +17,25 @@
     <dependency>
       <groupId>org.openjfx</groupId>
       <artifactId>javafx-controls</artifactId>
-      <version>12.0.1</version>
+      <version>12.0.2</version>
     </dependency>
     <!-- Uncomment for cross-platform fat jar
     <dependency>
         <groupId>org.openjfx</groupId>
         <artifactId>javafx-graphics</artifactId>
-        <version>12.0.1</version>
+        <version>12.0.2</version>
         <classifier>win</classifier>
     </dependency>
     <dependency>
         <groupId>org.openjfx</groupId>
         <artifactId>javafx-graphics</artifactId>
-        <version>12.0.1</version>
+        <version>12.0.2</version>
         <classifier>linux</classifier>
     </dependency>
     <dependency>
         <groupId>org.openjfx</groupId>
         <artifactId>javafx-graphics</artifactId>
-        <version>12.0.1</version>
+        <version>12.0.2</version>
```

Comment:
As stated before, these versions can move to maven properties to make future changes easier.